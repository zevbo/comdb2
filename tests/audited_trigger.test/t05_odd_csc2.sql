CREATE TABLE bx1155 { schema { cstring named[11] } }$$

CREATE LUA AUDITED TRIGGER omar ON (TABLE bx1155 FOR INSERT) 

INSERT INTO bx1155 VALUES("0123456789")

SELECT sleep(1)

SELECT * FROM "$audit_bx1155"

DROP LUA TRIGGER omar 
DROP TABLE "$audit_bx1155"
DROP TABLE bx1155