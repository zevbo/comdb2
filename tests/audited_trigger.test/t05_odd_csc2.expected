[CREATE TABLE bx1155 { schema { cstring named[11] } }] rc 0
[CREATE LUA AUDITED TRIGGER omar ON (TABLE bx1155 FOR INSERT)] rc 0
(rows inserted=1)
[INSERT INTO bx1155 VALUES("0123456789")] rc 0
(sleep(1)=1)
[SELECT sleep(1)] rc 0
(new_named='0123456789')
[SELECT new_named FROM "$audit_bx1155"] rc 0
[DROP LUA TRIGGER omar] rc 0
[DROP TABLE "$audit_bx1155"] rc 0
[DROP TABLE bx1155] rc 0
